### PR TITLE
devtools helpers extended

### PR DESCRIPTION
**Pull request by NikVolf**
- stop guard
- random strings

### CODE REVIEW

File: devtools/src/stop_guard.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Random path
```
Comment (rphmeier) [inline devtools/src/stop_guard.rs]: leftover doc comment

File: devtools/src/random_path.rs [hunk 2]
```diff
@@ -26,7 +26,11 @@ pub struct RandomTempPath {
 }
 
 pub fn random_filename() -> String {
-	(0..8).map(|_| ((random::<f32>() * 26.0) as u8 + 97) as char).collect()
+	random_str(8)
+}
+
+pub fn random_str(len: usize) -> String {
+	(0..len).map(|_| ((random::<f32>() * 26.0) as u8 + 97) as char).collect()
```
Comment (rphmeier) [inline devtools/src/random_path.rs]: random_alpha_lowercase might be a more descriptive name

Comment (NikVolf) [inline devtools/src/random_path.rs]: better keep it small until we have just one

Comment (gavofyork) [inline devtools/src/random_path.rs]: `random<f32>`? really? what about plain old `random<u8> % 26`?

Comment (arkpar) [inline devtools/src/random_path.rs]: It is not strictly uniform, not that it matters in this function

Comment (NikVolf) [inline devtools/src/random_path.rs]: it is not used anywhere except tests and it's been there for ages
it is not about this pr

**Pull request merged**